### ISSUE TITLE
Add button to silently send the current URL to wallabag in the adress bar.

### ISSUE THREAD

**Issue by N07070** [wallabag/firefox-ext-v1#19]
Hello.

I recently switched from Pocket to Wallabag, it's wayyy better.
But there is one feature I really loved about pocket: the small button in the address bar on which I click to send the current URL to my wallabag. 

Could this be implemented ? Also, it should be silent to never distract the user from what he is doing.

Cheers;

**Comment by tcitworld** [comment]
You mean the one that landed in Firefox 38 by default ? A *kind of same* button comes with next version of wallabag.

**Comment by N07070** [comment]
Kind of. Also, it should not open a window when saving a URL unless there is a issue, right ?

**Comment by tcitworld** [comment]
There's a session issue, yes.

**Comment by N07070** [comment]
So all I requested will be implemented in the next update of the Firefox Extension ? In how much time ?

**Comment by tcitworld** [comment]
It's not an Firefox extension, it uses Firefox Services [kind of like](https://activations.cdn.mozilla.net/fr/pocket.html) Pocket does. You will have a link to use it inside the config page of wallabag.

I hope to deliver next version before the end of the week, or else it will be next month.

**Comment by N07070** [comment]
Wow that's cool. Do you need any help ?

**Comment by tcitworld** [comment]
Any help is always welcomed. Please have a look at what we plan for wallabag v2 and how apps and extensions will change. 
I would also say there's work for any kind of skills.

**Comment by N07070** [comment]
Link ? :smile:

**Comment by tcitworld** [comment]
Well, here are some little content, I'm sorry I couldn't find more : 
https://github.com/wallabag/wallabag/issues/687
https://github.com/wallabag/wallabag/issues/1182
https://github.com/wallabag/wallabag/pull/1049

You may also get interested by all the issues with the *feature* label :wink: https://github.com/wallabag/wallabag/issues?q=is%3Aopen+is%3Aissue+label%3AFeature

And also, not related with php : All the third party apps and the templates for the themes.

**Comment by IzzySoft** [comment]
Chiming in here, albeit a bit late: A long-time user of Pocket, the "integration" with FF and dropping the addon (and a bunch of functionality along), I'm also [looking for alternatives](http://softwarerecs.stackexchange.com/q/27747/185). It seems *Wallabag* is the best candidate.

But back to topic: that very same icon *inside the address bar* on can use to add an URL to "the bag" with one click, it should also indicate if the page currently opened is already in the bag. With the pocket addon, it does so changing its color from gray to red. Does Wallabag have something similar (or will the new addon/service/whatever have it)?

**Comment by tcitworld** [comment]
wallabag server v2 will allow that kind of stuff, totally.

**Comment by IzzySoft** [comment]
@tcitworld Thanks! That's great news. So as soon that's out, I'll switch over (guess not only me). Looks like that (in)famous "integration" backfires for Pocket/Firefox (at least partially). Thanks to them finally giving me the kick for more privacy *LOL* – and of course to you giving the chance for that!

Looking forward to v2 then. As I'm subscribed here, I guess I'll get a notice beforehand (when this issue gets closed) :smiley_cat:

**Comment by tcitworld** [comment]
>  Looks like that (in)famous "integration" backfires for Pocket/Firefox (at least partially).

They are now moving Pocket to an external addon (still included by default though) but this should allow us to deliver the same features than them (well, you know, one day...).

**Comment by IzzySoft** [comment]
> They are now moving Pocket to an external addon (still included by default though) but this should allow us to deliver the same features than them (well, you know, one day...).

Figured that much from your "summer comments" (except for the "external addon" part – but that Wallabag can use the same API then). Moving Pocket to an external addon, even though included by default, let's me hope it can be removed easily then.

If Wallabag offers what I described above (basically, what Pocket did before the "integration"), and Pocket offers more (one day), I still will move to Wallabag and will be *very* content with that. I have what I really need then – and no "nice-to-have" can beat the added plus of privacy and having full control of your data.

Big thumbs up to you, Thomas (and to your team)!
